Tools/autotest/common.py

```diff
@@ -2150,19 +2150,14 @@ def get_sim_parameter_documentation_get_whitelist(self):
             "SIM_ADSB_COUNT",
             "SIM_ADSB_RADIUS",
             "SIM_ADSB_TX",
-            "SIM_ARSPD2_FAIL",
             "SIM_ARSPD2_FAILP",
```

Comment:
For ex, ARSPD2_FAIL exists in common.py but doesn't explicitly exist in SITL.cpp